examples/direct_fidelity_estimation.py

```diff
@@ -29,27 +30,44 @@
 import cirq
 
 
-def build_circuit():
-    # Builds an arbitrary circuit to test. The circuit is non Clifford to show
-    # the use of simulators.
-    qubits = cirq.LineQubit.range(3)
-    circuit = cirq.Circuit(
-        cirq.Z(qubits[0])**0.25,  # T-Gate, non Clifford.
-        cirq.X(qubits[1])**0.123,
-        cirq.X(qubits[2])**0.456)
+def build_circuit(
+) -> Tuple[cirq.Circuit, List[cirq.Qid], Optional[cirq.CliffordState]]:
+    # Builds an arbitrary circuit to test. If the circuit is Clifford, then it
+    # also returns a state from which we can get the stabilitzers. If the
+    # circuit is non Clifford, the state is None.
+    qubits: List[cirq.Qid] = [qubit for qubit in cirq.LineQubit.range(3)]
+    circuit: cirq.Circuit = cirq.Circuit()
+    clifford_state = cirq.CliffordState(
+        qubit_map={qubits[i]: i for i in range(len(qubits))})
+    is_clifford = True
+    gates = [
+        cirq.Z(qubits[0]),
+        cirq.X(qubits[1]),
+        cirq.X(qubits[2]),
+    ]
+    for gate in gates:
+        circuit.append(gate)
+        if is_clifford:
```

Comment:
I'm confused by the logic here. You're explicitly constructing a clifford circuit so this will always be `True`

Reply:
That was stemming from my buggy code that added a measurement gate, which in turn made iteration through the circuit operation not translatable to a Clifford state because the last gate isn't Clifford.

Actually, this was a bug from previous PRs (sorry), which made the fidelity values not correct, I think. As a check, I reran the job without any Clifford acceleration and without noise. I think under these settings the fidelity should be exactly 1.0. It was, however, exactly 0.125, so I added fixed this by adding the missing coefficient. 

Thanks for the carefully review and catching this.